moto/mediastore/urls.py

```diff
@@ -6,7 +6,6 @@
 ]
 
 
-
 url_paths = {
-    '{0}/$': MediaStoreResponse.dispatch,
+    "{0}/$": MediaStoreResponse.dispatch,
```

Comment:
Hm.. I'm not sure this is enough. You might have to explicitly list the URLs for the supported endpoints here (from the AWS documentation)